onnxruntime/contrib_ops/rocm/bert/skip_layer_norm.h

```diff
@@ -19,6 +19,7 @@ class SkipLayerNorm final : public RocmKernel {
 
  private:
   float epsilon_;
+  bool tuning_;
```

Comment:
We must not store this enabler, instead, we shall passing this bool directly from OpKernel function call.

Reply:
Can you elaborate a bit on the reason for this?

Reply:
Because we always bind thoes state to EP or some EP related objects. Then they can be accessed from method of `RocmKernel`. This avoid us from scattering all thoes states everywhere.

Reply:
Sounds Good. May change it when `session option` is down, same as `TransformerOptions` issue.